the-blue-alliance-ios/View Elements/Match/MatchSummaryView.swift

```diff
@@ -115,12 +133,26 @@ class MatchSummaryView: UIView {
             blueScoreLabel.font = winnerFont
         }
     }
+    
+    private func addRPToView(stackView: UIStackView, rpCount: Int) {
+        for _ in 0..<rpCount {
+            let rpLabel = label(text: "•", isBold: true)
+            stackView.insertArrangedSubview(rpLabel, at: 0)
```

Comment:
I think we can do `addArrangedSubview` here - I'm actually fairly certain the way we're inserting the team labels right now is causing a bug that I've worked around improperly...